### DISCUSSION TITLE
After changing "ST4" to "ST6", the output result is incorrect.

### DISCUSSION THREAD

**Discussion by nYifeiLee** [NOAA-EMC/WW3#1250] [Q&A]
Hi everyone！
After I changed the switch, I got an error with the field data and point data that was output.At the beginning, I turned on the "ST4" switch and got a good result for field data and point data.But after I changed "ST4" to "ST6" and "FLX0" to "FLX2", I calculated the same range, used the same water depth and wind field data, but the output field data result and point data result were both 0.I recompiled WW3 after changing the switch and I was confused by this result because I was a beginner to WW3. The following is a screenshot of the switch before and after, I hope you can give me some suggestions.
Before：
![Ubuntu 22 - VMware Workstation 2024_6_18 18_36_48](https://github.com/NOAA-EMC/WW3/assets/167869457/11e8f7ef-eddb-4d4a-9541-a6379e94c3ec)
After：
![Ubuntu ST6 - VMware Workstation 2024_6_18 18_34_24](https://github.com/NOAA-EMC/WW3/assets/167869457/84f37964-4856-4f5a-bdb7-e8d3cfcbaf77)

**Comment by mickaelaccensi** [comment]
Hi,
you should look into the log files and output files generated when running the different ww3 programs, there is probably an issue because ST6 should works as. Please also have a look and try some regtests using ST6
Best,

**Reply by nYifeiLee** [reply]
Thank you for your reply. Yes, I think it works as well, but I don't know why this error occurs. And after the switch is replaced, the MPI cannot be used normally.

**Comment by adegroodt** [comment]
Hi Yfei Lee,

I have the same issue, did you manage to resolve it ?

**Reply by nYifeiLee** [reply]
Not yet, I'm exploring ways to fix it.

**Comment by mickaelaccensi** [comment]
could you confirm that you are using the latest version of the github repo ? 
Can you try to run some regression tests with ST6, for example : 
-ww3_ts1/input/switch_ST6
-mww3_test_05/input/switch_ST6_PR3_UQ_MPI

You can also provide the log files of you run to check if something went wrong

**Reply by adegroodt** [reply]
Hi,
I'm trying to get the best results for Whitecap coverage, with ST4 it works fine but I have tried both regression tests you suggested and I get NaN value for "wcc".
